starter/script.js

```diff
@@ -1 +1,33 @@
 // put code here!
+var tantrumFree = function() {
+  console.log("**************************")
+}
+console.log ("You've overslept one morning and realized you have 25 minutes to get yourself and your kindergartener out the door")
+var wakeUpInput = prompt("You can either jolt out of bed or push the snooze button") (Please enter 'j' or 's')
+
+if (wakeUpInput) == "j"{
+  console.log("You jolt out of bed and begin dressing your kindergartener, when she decides she doesn't want to wear your option but rather her polka-dot pink Doc McStuffin shirt with her green striped duck pants."
+}
+var (reponseToToddler) = prompt("You realize that this very moment could make or break her blossoming kindergarten reputation" and decide to \n('a') - argue with her. We're talking reputation here people!...hers and yours!\n('g') - give in, toddlers have no reputation.  This may actually increase her self-confidence, while the other parents question your parenting skills\n('n')- negotiate with her.  She may know the latest trends that you may not be hip to and needs just a little guidance.")
```

Comment:
Note that this line has a syntax error, that's because you've ended the quote marks around the string after "blossoming kindergarten reputation". Be cause of this, JS is trying to treat the following words "and decide to ..." as *code* rather than text in your program. This would almost certainly throw off the rest of the program.